app/controllers/projects_controller.rb

```diff
@@ -1,8 +1,5 @@
 class ProjectsController < ApplicationController
   def show
-    project = Project.find(params[:id])
-    @project_name = project.name
-    @project_material = project.material
-    @project_theme = project.challenge.theme
+    @project = Project.find(params[:id])
   end
```

Comment:
I think I should have titled this commit "Refactor projects_controller show action."